typescript/utils/src/validator.ts

```diff
@@ -1,47 +1,64 @@
-import { BigNumberish, ethers } from 'ethers';
+import { ethers } from 'ethers';
 
 import { types, utils } from '@abacus-network/utils';
 
-export class Validator {
+import { Checkpoint } from './types';
+
+export class BaseValidator {
```

Comment:
Can we get some docs on these calsses to better explain what their intents are, because reading this at first I was thinking this was for the act of validating things, but it looks like it is actually for the validators the noun, so a bit confusing.

Reply:
definitely